contrib/pg_tde/t/pgtde.pm

```diff
@@ -56,6 +56,21 @@ sub pgtde_init_pg
     return $pg_node;
 }
 
+sub psql
+{
+    my ($node, $dbname, $sql) = @_;
+
+    my (undef, $stdout, $stderr) = $node->psql($dbname, $sql, extra_params => ['-a']);
+
+    if ($stdout ne '') {
+        append_to_file($stdout);
+    }
+
+    if ($stderr ne '') {
+        append_to_file($stderr);
+    }
+}
+
 sub append_to_file
 {
```

Comment:
Suggestion, Wouldn't changing it to `append_to_result_file `be better?

Reply:
Yeah, that is a better name.